client/gutenberg/editor/style.scss

```diff
@@ -108,4 +108,35 @@ $gutenberg-theme-toggle: #11a0d2;
 			z-index: 100000; // Above WP toolbar.
 		}
 	}
+
+	.edit-post-header-toolbar__back.is-borderless {
```

Comment:
Restoring this from `gutenberg/editor/edit-post/components/header/header-toolbar/style`. Originally introduced in https://github.com/Automattic/wp-calypso/pull/26809/.